cartographer/mapping/internal/2d/ray_to_pixel_mask_test.cc

```diff
@@ -65,7 +65,7 @@ TEST(RayCastingTest, AxisAlignedY) {
   EXPECT_TRUE(ray[2].isApprox(Eigen::Array2i({1, 3})));
```

Comment:
You are checking for the exact sequence, which is strict. It would be slightly better to use
https://github.com/google/googletest/blob/master/googlemock/docs/CheatSheet.md#container-matchers
that ignore the ordering. For this, you need to define a new matcher for Array2i similar to what's done in cartographer/sensor/compressed_point_cloud_test.cc:43